js/components/Expenditure.js

```diff
@@ -179,31 +179,32 @@ class BudgetExpenditure extends HTMLElement {
 
   ex_approve(expenditureID){
     patchData(`api/expenditure/${expenditureID}/state/1`).then( (response) => {
```

Comment:
This merge conflict was resolved incorrectly. The updated should be reloadProfileData(this.talent_id).